src/main/java/Circle.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Random;
+
+public class Circle extends Figure implements Draw {
```

Comment:
```suggestion
public class Circle extends Figure {
```
 Figure implements Draw, тому цей тип перейде автоиматично усім наслідникам